modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -221,31 +222,37 @@ private List<String> _collectSassFiles(String dirName, File baseDir)
 	}
 
 	private long _getNewestModifiedTime(String baseDir, String[] fileNames) {
-		return Stream.of(
-			fileNames
-		).map(
-			fileName -> Paths.get(baseDir, fileName)
-		).map(
-			FileUtil::getLastModifiedTime
-		).max(
-			Comparator.naturalOrder()
-		).orElse(
-			Long.MIN_VALUE
-		);
+		List<Long> lastModifiedTimes = TransformUtil.transform(
+			TransformUtil.transformToList(
+				fileNames, fileName -> Paths.get(baseDir, fileName)),
+			FileUtil::getLastModifiedTime);
+
+		if (lastModifiedTimes.isEmpty()) {
+			return Long.MIN_VALUE;
+		}
+
+		Comparator<Long> comparator = Comparator.naturalOrder();
+
+		lastModifiedTimes = ListUtil.sort(
```

Comment:
@slnn Additionally, please just use `lastModifiedTimes.sort()` so that you can avoid introducing `portal-kernel` dependency. The list returned by `TransformUtil` must be a modifiable list.